docs/user_guide/preemption.md

```diff
@@ -228,4 +228,31 @@ In this example, two imbalances are observed:
 | `rt.ten-a.queue-2` | 0                          | 0                         |
 | `rt.ten-b`         | 15                         | 10                        |
 | `rt.ten-b.queue-3` | 15                         | 10                        |
-| `rt.sys`           | 0                          | 10                        |
\ No newline at end of file
+| `rt.sys`           | 0                          | 10                        |
+
+### Redistribution of Quota and Preemption Storm
+
+#### Redistribution of Quota
+
+Setting up guaranteed resources for the queue present at a higher level in the whole queue hierarchy helps to re-distribute the quota among different groups especially when workloads of the same priority run in different groups. Unlike the default scheduler, Yunikorn preempts even the workloads of the same priority to free up resources for pending workloads who deserve to get the resources as per guaranteed quota. At times, one needs this kind of queue set up in a real production cluster for redistribution.
```

Comment:
drop 'whole' from "the whole queue"
start new sentence at "especially"
Yunikorn with a capital K
drop 'even the' from "even the workloads"
add "the queues' " to "as per the queues' guaranteed quota"
set up is one word "setup"